images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -115,7 +114,9 @@ $markdown += New-MDList -Style Unordered -Lines @(
     (Get-GoogleCloudSDKVersion)
 )
 
-$markdown += New-MDHeader "Rust packages:" -Level 3
+$markdown += New-MDHeader "Rust" -Level 3
+$markdown += New-MDParagraph -Lines "Version: $(Get-RustVersion)"
+$markdown += New-MDHeader "Packages:" -Level 3
```

Comment:
We don't need `:` here

Reply:
Also I think `Packages` section should have level higher than title

Reply:
Done.